docs/examples/MarrmotM01.ipynb

```diff
@@ -2,196 +2,142 @@
  "cells": [
```

Comment:
<p>Can comments be moved to markdown cell above it? It is harder to miss text in markdown cell then a grayed out comment.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/eWaterCycle/ewatercycle/pull/158/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Comments and headers are revised. </p>